packages/test/functional-tests/src/test/ddsHandleEncoding.spec.ts

```diff
@@ -102,25 +110,72 @@ describe("DDS Handle Encoding", () => {
 			},
 			[handle.absolutePath] /* expectedHandles */,
 		),
+		createTestCase(
+			new SharedStringFactory(),
+			(dds) => {
+				dds.insertMarker(0, ReferenceType.Simple, { marker: handle });
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
+		createTestCase(
+			new SharedMatrixFactory(),
+			(dds: SharedMatrix) => {
+				dds.insertRows(0, 1);
+				dds.insertCols(0, 1);
+
+				dds.setCell(0, 0, handle);
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
+		createTestCase(
+			new TreeFactory({}),
+			(dds: ITree) => {
+				const builder = new SchemaFactory("test");
+				class Bar extends builder.object("bar", {
+					h: builder.optional(builder.handle),
```

Comment:
Would any other structural pieces represent distinct test cases, or is this sufficient?

Reply:
I think this combined with the handle tests in the shared-tree package directly give sufficient coverage.